src/cache/cache_las.c

```diff
@@ -1002,7 +1008,13 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
     __wt_las_cursor(session, &cursor, &session_flags);
     WT_ASSERT(session, cursor->session == &session->iface);
     __las_set_isolation(session, &saved_isolation);
-    WT_ERR(__wt_txn_begin(session, NULL));
+
+    /*
+     * Open a transaction with snapshot isolation. The LAS cursor moves it's position to either prev
+     * or next many times during the sweep process, so make sure that the cursor doesn't see newer
+     * updates.
+     */
+    WT_ERR(__wt_txn_begin(session, txn_cfg));
```

Comment:
No. The sweep server during the processing of a sweep, it shouldn't see other inserts that are happened to the LAS in parallel. As we are cleaning the previous records based on the next record, just want to avoid scenarios where it can see a completely different record.